packages/documentation/src/pages/about-mdx.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: About MDX
+---
+
+## About MDX
+
+### What is MDX?
+We will be using the .mdx format to create our documents.
+MDX is a format that lets you seamlessly use JSX in your Markdown documents.
+You can import components, like interactive charts or notifs, and export metadata.
+This makes writing long-form content with components a blast 🚀
+
+[https://mdxjs.com/](https://mdxjs.com/)
+
+### Example
+Here is a great page of examples of using MDX
+
+- http://dump.jamesknelson.com/mdxc-playground.html
+
+### Findings on things you can do with MDX and gotchas
+This is just documentations on finding of using MDX in our projects. Please feel
+free to add to this document if you find anything else.
+
+#### Inline components
+You can create components incline inside your MDX files. Below is an example
+of how to do it. You will need to export it for it to work. You can write both
+function based components and class based components. The reason why you might
+use this is if your component updates the parent state. In the example you need
```

Comment:
`In the example you need` -> maybe change the subject of this to the example: `This example uses state to toggle the drop down` unless we're directing them to implement this example?